### ISSUE TITLE
"Nx generate (ui)" from explorer prefills invalid path which results in error

### ISSUE THREAD

**Issue by stefan-schweiger** [nrwl/nx-console#1061]
When I right click on a specific folder in the explorer because I want to generate a library with "Nx generate (ui)" in this directory it always fills out the absolut path from the root, but this value is actually invalid for the ng generate command:

```
> Executing task:
ng generate @nrwl/angular:library --name=expert-profile --directory=/libs/shared/ui --no-interactive --dry-run
<

Project name "-libs-shared-ui-expert-profile" is not valid. New project names must start with a letter,
and must contain only alphanumeric characters or dashes. When adding a dash the segment after the
dash must also start with a letter.
-libs-shared-ui-expert-profile
```

When I manually change the value to `shared/ui` it works. I think changing the behavior of `ng generate` is probably not an option. So I think nx console should set a path which doesn't result in error.

Nx Console v17.1.0
nx 12.0.2

**Comment by sandikbarr** [comment]
That bug did get introduced recently with the addition that saves the workspace config path. But hmm, it should have been [fixed in Nx Console v17.1.0](https://github.com/nrwl/nx-console/commit/2e6449bc23d5a4c52a5bede26d5c41ab52c47817)

We only recently added support to alias the directory property to path to support that as well. Do you also get a leading '/' on generators with a path property? An example using path would be `@schematics/angular:component`

**Comment by stefan-schweiger** [comment]
@sandikbarr When I try to generate an Angular component I also get the leading `/libs`  but in this case it works without a problem:

```
> Executing task:
ng generate @schematics/angular:component --name=test --project=shared-ui-expert-profile --path=/libs/shared/ui/expert-profile/src/lib --no-interactive --dry-run
<

CREATE libs/shared/ui/expert-profile/src/lib/test/test.component.html
CREATE libs/shared/ui/expert-profile/src/lib/test/test.component.spec.ts       
CREATE libs/shared/ui/expert-profile/src/lib/test/test.component.ts
CREATE libs/shared/ui/expert-profile/src/lib/test/test.component.scss
UPDATE libs/shared/ui/expert-profile/src/lib/shared-ui-expert-profile.module.ts
```

**Comment by sandikbarr** [comment]
Thinking this could be a Windows issue I didn't account for in that [first change](https://github.com/nrwl/nx-console/commit/2e6449bc23d5a4c52a5bede26d5c41ab52c47817). I put in a fix in #1063 to resolve it if that's the case.
